CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [6.1.0] - TBD
 ### Changed
+- Maven Group Id changed from `com.klarna` to `io.github.hiverunner`.
```

Comment:
Normally I'd consider this a breaking change and would increase the major version number, but since we also need to make this exact same change in the `hive-2.x` branch where we can't change the major version number I thought I'd leave this as 6.x and only do a major bump here when there is a backwards-incompatible change that relates to Hive 3.x. Happy to hear other opinions though.